libs/ledger-live-common/src/exchange/swap/index.ts

```diff
@@ -27,9 +27,13 @@ import submitKYC from "./submitKYC";
 export const operationStatusList = {
   finishedOK: ["finished"],
   finishedKO: ["refunded"],
-  pending: ["pending", "onhold", "expired", "unknown"],
 };
 
+// A swap operation is considered pending if it is not in a finishedOK or finishedKO state
+export const isSwapOperationPending: (status: string) => boolean = (status) =>
```

Comment:
As you refactored the call to `operationStatusList`, shouldn't we remove totally this hashmap?
We could have a have a simplier isSwapOperationPending function by keeping the possible status:
```ts
type Status = "finished" | "refunded" | "pending" | "onhold" | "expired";

export function isSwapOperationPending(status: Status) {
  return status !== "finished" || status !== "refunded";
}
```

Reply:
Not really, the hashmap is still used for the OK and KO cases